audiorecorder/src/main/java/org/odk/collect/audiorecorder/DaggerSetup.kt

```diff
@@ -91,25 +79,14 @@ internal open class AudioRecorderDependencyModule {
         }
     }
 
-    @Provides
-    @Singleton
-    open fun providesRecordingRepository(): RecordingRepository {
-        return RecordingRepository()
-    }
-
     @Provides
     open fun providesScheduler(application: Application): Scheduler {
         return CoroutineScheduler(Dispatchers.Main, Dispatchers.IO)
     }
-}
-
-internal fun RobolectricApplication.clearDependencies() {
-    _component = null
-}
 
-internal fun RobolectricApplication.setupDependencies(module: AudioRecorderDependencyModule) {
-    _component = DaggerAudioRecorderDependencyComponent.builder()
-        .application(this)
-        .dependencyModule(module)
-        .build()
+    @Provides
+    @Singleton
+    internal open fun providesRecordingRepository(): RecordingRepository {
```

Comment:
Marking this `internal` means that a dependent application cannot override this dependency (we don't want them changing this probably).